src/main/java/Duke.java

```diff
@@ -5,114 +5,167 @@
 
 public class Duke {
 
+    public static final String GREETING_MESSAGE =
+            "Hello! I'm Duke" + "\n"
+            + "What can I do for you?" + "\n"
+            + "Tell me your plan!";
+    public static final String LIST_DISPLAY_MESSAGE = "Here are the tasks in your list:";
+    public static final String MESSAGE_RECEIVED_MESSAGE = "Got it. I've added this task:";
+    public static final String SPECIFY_TIME_WARNING = "Please specify time!";
+    public static final String ILLEGAL_CMD_WARNING = "No such command! Please try again!";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
     public static final String HORIZONTAL_LINE = "---------------------";
-    public static final String DONE_CMD = "done ";
-    public static final String BYE_CMD = "bye";
-    public static final String TODO_CMD = "todo ";
-    public static final String DEADLINE_CMD = "deadline ";
-    public static final String EVENT_CMD = "event ";
-    public static final String ILLEGAL_CMD_WARNING = "No such command! Please try again";
-    public static final String LIST_CMD = "list";
     public static final String DDL_CMD_SEPARATOR = "/by";
     public static final String EVENT_CMD_SEPARATOR = "/at";
+    public static final String DONE_CMD = "done";
+    public static final String BYE_CMD = "bye";
+    public static final String TODO_CMD = "todo";
+    public static final String DEADLINE_CMD = "deadline";
+    public static final String EVENT_CMD = "event";
+    public static final String LIST_CMD = "list";
```

Comment:
I like these public static final Strings!

Reply:
HAHA Tks

Reply:
I really like your coding styles. It is super clear and easy to understand and to read as well. They weren't any magic numbers and the refactoring is awesome!